qbx_management.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS `player_jobs_activity` (
+  `id` int UNSIGNED NOT NULL AUTO_INCREMENT,
+  `citizenid` varchar(50) NOT NULL,
```

Comment:
Should this be a foreign key to the players table?

Reply:
It would protect against an invalid citizenid, and allow for automatic deletion of this data if the player is deleted? What's the reason not to do it?

Reply:
I'd prefer it, yes.